package.json

```diff
@@ -36,6 +35,7 @@
     "coveralls": "^2.0.0",
     "mocha": "^3.0.0",
     "nyc": "^10.0.0",
+    "redux": "^3.7.2",
```

Comment:
isn't this a dependency rather than dev dependency?

Reply:
We only import types from it, not any actual functionality. I looked at the build output file as well, it contains nothing from redux, nor any imports.